railgun/test/integration/ingress_bulk_test.go

```diff
@@ -0,0 +1,109 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/pkg/annotations"
+	"github.com/kong/kubernetes-testing-framework/pkg/utils/kubernetes/generators"
+)
+
+const testBulkIngressNamespace = "ingress-bulk-testing"
+
+func TestIngressBulk(t *testing.T) {
+	ctx := context.Background()
+
+	t.Logf("creating namespace %s for testing", testBulkIngressNamespace)
+	namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testBulkIngressNamespace}}
```

Comment:
You can leverage the function CreateNamespace under performance direcotry

Reply:
I appreciate that suggestion, but at the moment that exists under a different build tag so I would need to move the function which is a heavier touch than I was hoping with this PR, and beyond that it has additional functionality I don't see as necessary (just a basic namespace is all that's needed here at the moment).

Reply:
Can you please move the CreateNamespace function to pkg/util directory, and all of us can leverage from the library ?

Reply:
There's a follow up issue now for the namespace management functionality in our tests so that we can move this stuff around after 2.0 and there's more time: https://github.com/Kong/kubernetes-ingress-controller/issues/1574